pages/index.js

```diff
@@ -62,8 +62,12 @@ const Subtitle = styled.h2`
 export default function Home() {
   return (
     <>
+      <Head>
+        <title>CBASP Companion</title>
+        <meta charSet="UTF-8"></meta>
+      </Head>
       <Zoom duration={300}>
-        <TitleBox>
+        <TitleBox onClick>
```

Comment:
The `onClick` prop doesn't seem to do anything as it's not being used in the styled component.